examples/flava/native/README.md

```diff
@@ -0,0 +1,33 @@
+# Usage Instructions
+
+## Prerequisites
+
+- Installed PyTorch 1.13 or nightly 
+- `git clone https://github.com/facebookresearch/multimodal`
```

Comment:
just link to the library readme as first step and pip install on the flava requirements as second step? that should be enough?

Reply:
the examples/ folder isn't included if users install the library via pip:

https://github.com/facebookresearch/multimodal/blob/main/setup.py#L80

Reply:
you need to link to this https://github.com/facebookresearch/multimodal/blob/main/README.md#building-from-source